src/repositories/errors.ts

```diff
@@ -32,8 +32,8 @@ export class InsertionException extends RepositoryException {
 export class EntryNotFoundAfterInsertException<T extends BaseEntity> extends RepositoryException {
   constructor(entity: EntityTarget<T>) {
     super(
-      `The insertion of a new ${entity} failed, the created id was not found.`,
-      'InsertionException',
+      `The insertion of a new ${entity} failed, the created entry was not found.`,
+      'EntryNotFoundAfterInsertException',
```

Comment:
I've just remembered that we should probably be careful with this kind of exception.
Recently we had an issue where we were throwing too much information to the user, so Basile decided to throw only the Id of the entity.

I don't know what we're expecting the error message to be here, I feel like it's gonna be :
 `The insertion of a new [object Object] failed, the created entry was not found.` which doesn't make sense either if I'm right.

Reply:
I totally agree with you! Here the entity is not the instance of an BaseEntity, but just the class of the Entity. It is just used to display the Entity's name in the exception like:

"The deletion of class Item". But I just saw that it generate something like "The deletion of class Item extends typeorm_1.BaseEntity", so I have to fix that ^^.

Reply:
But I just saw that I effectively mad a mistake, so thanks!